### DISCUSSION TITLE
almi5 dataset no longer available?

### DISCUSSION THREAD

**Discussion by s-romeni** [simnibs/simnibs#244] [6. Getting started] [answered]
Hi all,

I am starting using SimNIBS and I am trying to replicate some published papers that use the almi5 dataset. While they say that the almi5 dataset is included in SimNIBS, I cannot find it among the datasets that can be accessed [here](https://simnibs.github.io/simnibs/build/html/dataset.html). Is that because it was available with older SimNIBS versions? Is there a way to get this dataset?

Thank you,

Simone

**Comment by TorgeW** [comment, marked as answer]
It is not available from our side anymore.

Sorry for the inconvenience.
